index.css

```diff
@@ -635,29 +652,26 @@ body {
 
 .form {
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
```

Comment:
- [ ] Kindly make sure your text and form section are aligned correctly as seen in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/206720620-05f8924c-df6d-441f-8776-7cb306aa4d6c.png)  |  ![image](https://user-images.githubusercontent.com/74814780/206720540-eb582db6-cc46-454c-9360-a57ac3f52936.png)